Add a way to retrieve battery level via HTTP

Deadolus opened the pull request:
**Description:**
The MQTT way of receiving data only seems to work if you have configured a MQTT backend. 
Added a template sensor to get the battery level with a HTTP Owntracks backend.
This way is tested on my Hometracks system.

**Pull request in home-assistant (if applicable):** 

## Checklist:

- [x ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

Deadolus committed:
Add a way to retrieve battery level via HTTP

The MQTT way of receiving data only seems to work if you have configured a MQTT backend. 
Added a template sensor to get the battery level with a HTTP Owntracks backend.
This way is tested on my Hometracks system.

frenck commented on source/_cookbook/track_battery_level.markdown:
```diff
@@ -41,3 +45,19 @@ sensor:
+- platform: template
+    sensors:
+      your_battery_sensor_name:
+        value_template: '{{ states.device_tracker.deviceid.attributes.battery_level }}'
```
We should not retrieve attributes like that, please use `state_attr('device_tracker.deviceid', 'battery_level')` instead.
More information: <https://www.home-assistant.io/docs/configuration/templating/#states>

frenck commented on source/_cookbook/track_battery_level.markdown:
```diff
@@ -41,3 +45,19 @@ sensor:
+#### HTTP
+
+If you have configured Owntracks to send reports to your Homeassistant instance via HTTP you can use a template sensor. 
+Replace deviceid with the set Device ID in Owntracks.
```
```suggestion
Replace `deviceid` with the set Device ID in Owntracks.
```

frenck commented on source/_cookbook/track_battery_level.markdown:
```diff
@@ -41,3 +45,19 @@ sensor:
+
+#### HTTP
+
+If you have configured Owntracks to send reports to your Homeassistant instance via HTTP you can use a template sensor. 
```
```suggestion
If you have configured Owntracks to send reports to your Home Assistant instance via HTTP you can use a template sensor. 
```

frenck reviewed (changes requested):
Thanks for the PR @Deadolus! I've left you some comments, could you please take a look? Thanks! 👍

Deadolus committed:
Apply direct suggestions from code review

Added the easy changes where @frenck gets attributed

Co-Authored-By: Franck Nijhof <frenck@frenck.nl>

Deadolus committed:
Applied suggestion to value_template 

Changed the way the value_template gets the battery_level as suggested by @frenck . 
Tested on my system to make sure it works.

Co-Authored-By: Franck Nijhof <frenck@frenck.nl>

Deadolus commented:
Thank you @frenck for the suggestions. 
I didn't know you're not supposed to get values the way I did. 
Thank you for teaching me something new.
I added in all your suggestions.

frenck reviewed (approved):
Thanks for adjusting @Deadolus! 👍 🎉
